src/invetica/test/util.clj

```diff
@@ -0,0 +1,6 @@
+(ns invetica.test.util
+  (:require [clojure.pprint :refer [pprint]]))
+
+(defn pprint-str
+  [x]
+  (with-out-str (pprint x)))
```

Comment:
I use this a lot, but not sure it warrants an entire library. If only my test runner were a little friendlier to work with.